go.mod

```diff
@@ -91,6 +87,7 @@ require (
 	github.com/Azure/go-ntlmssp v0.0.0-20221128193559-754e69321358 // indirect
 	github.com/AzureAD/microsoft-authentication-library-for-go v1.2.2 // indirect
 	github.com/DataDog/zstd v1.5.5 // indirect
+	github.com/IBM/sarama v1.41.2 // indirect
```

Comment:
Why still import sarama, it should be removed from the dependency?

Reply:
It is indirect, and tiflow depends on it